doc/hop.txt

```diff
@@ -806,6 +806,12 @@ below.
     Defaults:~
         `multi_windows = false`
 
+`excluded_filetypes`
+    Set the excluded filetypes to skip hint.
```

Comment:
This documentation is fairly incomplete. Please mention multi-window support because otherwise it doesn’t really make sense for people using Hop without `*MW`.

Reply:
Thank your review. I'll complete the documentation.